src/tests/mem/page_allocator.c

```diff
@@ -0,0 +1,30 @@
+/**
+ * @file
+ *
+ * @brief
+ *
+ * @date 12.07.2018
+ * @author Filipp Chubukov
+ */
+
+#include <embox/test.h>
+#include <mem/page.h>
+#include <mem/heap.h>
+#include <mem/phymem.h>
+#include <util/binalign.h>
+
+EMBOX_TEST_SUITE("page_alloc_init test");
+
+TEST_CASE("Init page_allocator with one page") {
+	struct page_allocator *allocator;
+
+	extern char _reserve_end;
+	char *const phymem_alloc_start = (char *)
```

Comment:
It is not a very good idea to use phymem_alloc_start, because of this address is already in use by phymem_allocator. A better way is to prealloc some memory region with phymem_alloc() and then initialize page allocator on this new region:

ptr = phymem_alloc(pages_count);
page_allocator_init(ptr,...);